egs/chime6/asr1/conf/train.yaml

```diff
@@ -0,0 +1,46 @@
+# minibatch related
```

Comment:
Can you also please add the transformer transducer configuration from your paper in a different file? (not the default option, just in case if someone wants to try it)

Reply:
Thanks for your suggestion!
I'll do it.